docs/setup.md

```diff
@@ -83,9 +83,13 @@ After installation of the **Outbreak Response Framework** solution pack, run the
         - *High*
         - *Medium*
 
-    - **Outbreak Alert Update Notification**:  Specify email addresses authorized to receive outbreak updates. You can specify multiple email addresses separated by a comma (`,`).
+    - **Outbreak Alert Update Notification**:  Specify email addresses authorized to receive outbreak updates. You can specify multiple email addresses separated by a comma.
 
-8. Click **Finish** to complete the configuration process.
+8. **Summary**
+
+    Click the button **Ingest Now** to install the outbreak response solution packs of the severity selected on the previous screen.
+
+    Click **Finish** to complete the configuration process.
```

Comment:
You will need to elaborate more on what things happen in the background as part of the finish page and overall configuration wizard like it creates 1,2,3 schedules for xyz things, stores hunt tools details and its param in key-store, etc.

When can user expect the installation of OBR SPs starts, same for invitation PBs, etc.

Please collaborate with Sujit for completing this section.